### ISSUE TITLE
Unable to navigate Popover content via Android TalkBack

### ISSUE THREAD

**Issue by dcalhoun** [WordPress/gutenberg#70277]
### Description

When leveraging [Android TalkBack](https://support.google.com/accessibility/android/answer/6006598?hl=en) to navigate a popover, the popover is unexpectedly dismissed when swiping to move to the next item in the popover. This occurs for both either navigating continuously (swipe left/right) or navigating via reading controls (swipe up/down). 

It's possible this may relate to https://github.com/WordPress/gutenberg/issues/22014. 

### Step-by-step reproduction instructions

1. Open the editor. 
2. Enable Android TalkBack. 
3. Select a text block (Paragraph). 
4. Move focus to the alignment button. 
5. Double tap to activate the button. 
6. Attempt to swipe right. 

### Screenshots, screen recording, code snippet

https://github.com/user-attachments/assets/b5a09c44-c5b6-4a43-a1e8-1412b4321d68

### Environment info

- WordPress 6.8
- Samsung Galaxy S20
- Android 13

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [x] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by jorgefilipecosta** [comment]
I was able to replicate the issue, not sure yet what the fix could be.
